### PR TITLE
Info and schedule about hack days

### PR DESCRIPTION
**Pull request by wtbarnes**
It's a little light on the details. Not sure how much information we want to make publicly available on the webpage? There's a link to the Riot chat and presumably we'll post all links there.

### CODE REVIEW

File: pages/2020/schedule.rst [hunk 1]
```diff
@@ -1,258 +1,66 @@
...
+  +------------+--------------------------------------------------------------------------------+
+  | Time (UTC) | Thursday (23 April)                                                            |
+  +============+================================================================================+
+  | 0:00       | Hacking continues                                                              |
+  +------------+--------------------------------------------------------------------------------+
+  | 7:50       | *Sophie Murray* - **Opening Remarks**                                          |
+  +------------+--------------------------------------------------------------------------------+
+  | 8:00       | *Brett Morris* - **State of the Package: Astropy**                             |
+  +------------+--------------------------------------------------------------------------------+
+  | 9:00       | *Speaker* - **Talk Title**                                                     |
```
Comment (SolarDrew) [inline pages/2020/schedule.rst]: Possibly replace this with something more official-sounding perhaps, "Talk TBD" or something? This seems a little too obviously placeholdery :P

File: pages/2020/index.rst [hunk 2]
```diff
@@ -43,6 +43,13 @@ All participants at the conference will be expected to follow the
 More details will be made available on this page closer to the week of the
 conference.
 
+Virtual Hack Day(s)
+###################
+
+In lieu of this year's in-person meeting, which was unfortunately cancelled due to the ongoing COVID-19 pandemic, we will be hosting a 24-hour virtual hack day from 22-24 April. This event is open to anyone who is interested in participating and not limited to those who were planning to attend the in-person meeting. Note that all participants are expected to follow the `code of conduct </code-of-conduct>`_ while participating in any hack day activities.
+
+The hack day will consist of a few talks from some of our invited speakers from the in-person meeting as well as two wrap-up sessions, to accommodate multiple time zones, in which participants can present the results of their hack. Depending on the total number of participants, the SOC may select only a few participants to present their hack day projects. The schedule `can be found here </2020/schedule>`_. More details will be posted in the `Matrix chat room <https://riot.im/app/#/group/+pyastro:openastronomy.org>`_ in the upcoming days.
```
Comment (SolarDrew) [inline pages/2020/index.rst]: We should add the link to the slack team here too if it's still live but I won't block merging this for it, we can add it in a separate PR if necessary.

**Review by SolarDrew — CHANGES_REQUESTED** [review]
Bar my one comment, looks good :+1: I think this level of detail is fine, as you say I think we want direct links to things in the chat rather than on the website.

File: pages/2020/index.rst (hunk 2 above)
Comment (wtbarnes) [inline pages/2020/index.rst]:
```suggestion
The hack day will consist of a few talks from some of our invited speakers from the in-person meeting as well as two wrap-up sessions, to accommodate multiple time zones, in which participants can present the results of their hack. Depending on the total number of participants, the SOC may select only a few participants to present their hack day projects. The schedule `can be found here </2020/schedule>`_. More details will be posted in the `Matrix <https://riot.im/app/#/group/+pyastro:openastronomy.org>`_ and `Slack <https://join.slack.com/t/pyastro/shared_invite/enQtOTU1NTA0MTIzMDMxLTUyOWJiYjQ0NmU0N2MxZDllMTFiMDRiNjViODgzMmE2ZWYzNmE5YjliNWUwM2NjYjI5Zjc0ZWUyZDMxYTQ2NTk>`_ rooms in the upcoming days.
```

File: pages/2020/schedule.rst (hunk 1 above)
Comment (wtbarnes) [inline pages/2020/schedule.rst]:
```suggestion
  | 9:00       | *TBD*                                                                          |
```

File: pages/2020/index.rst (hunk 2 above)
Comment (wtbarnes) [reply pages/2020/index.rst]: Looks like the Slack invite link is still working so I've pasted it here too.

**Pull request merged**